src/server/routes/new.route.js

```diff
@@ -48,104 +46,85 @@ const newRouter = () => {
-              Object.assign(req.session, req.session, { ...browserInfo });
-              const { country, local_council } = await getCouncilData(
-                req.params.lc
-              );
-              Object.assign(
-                req.session,
-                { country },
-                { lcName: local_council }
-              );
+      const localAuthorities = await localAuthoritiesCache.get();
 
-              if (MAINTENANCE_MODE_BLOCK_NEW === "true") {
-                logEmitter.emit(
-                  "functionSuccessWith",
-                  "Routes",
-                  "/new route",
-                  "Maintenance Mode (Block New Users) Active. Rendering page: /maintenance"
-                );
-                res.render("maintenance");
-              } else {
-                logEmitter.emit(
-                  "functionSuccessWith",
-                  "Routes",
-                  "/new route",
-                  "Session regenerated. Rendering page: /index"
-                );
+      // If it is legacy request to /new/local-authority, redirect to /new
+      if (
+        localAuthorities &&
+        localAuthorities.map((i) => i.local_council_url).includes(page)
+      ) {
+        logEmitter.emit(
+          "functionSuccessWith",
+          "Routes",
+          `/new/${page} route`,
+          "Redirect to /new"
+        );
+        res.redirect("/new");
+      }
 
-                var props = PropsGenerator(req);
-                res.render("index", { props });
-              }
-            } catch (err) {
-              logEmitter.emit("functionFail", "Routes", "/new route", err);
-              next(err);
-            }
-          });
-        } else {
-          // Save council to session if not yet there
-          if (!req.session.council) {
-            req.session.council = req.params.lc;
-          }
-          // Save the path config to the session if not yet there
-          if (!req.session.pathConfig) {
+      // If the requested page is the homepage, regenerate the session before rendering.
+      // (This wipes the user's data, allowing for a clean slate in the new registration)
+      if (page === "index") {
+        req.session.regenerate(async () => {
+          try {
             req.session.pathConfig = await getPathConfigByVersion(
               REGISTRATION_DATA_VERSION
             );
-          }
-          // Save the browser support to the session if not there yet
-          if (!req.session.isBrowserSupported) {
             const browserInfo = getBrowserInfo(req.headers["user-agent"]);
             Object.assign(req.session, req.session, { ...browserInfo });
-          }
-          // Save the country and council name to session if not yet there
-          if (!req.session.country) {
-            const { country, local_council } = await getCouncilData(
-              req.params.lc
-            );
-            Object.assign(req.session, { country }, { lcName: local_council });
```

Comment:
have we lost the country setting completely? I can't see where set now - should be when LA is determined